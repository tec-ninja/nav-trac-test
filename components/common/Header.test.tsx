import React from 'react'
import renderer from 'react-test-renderer'
import Header from './header'

it('Header renders correctly', () => {
    const tree = renderer
        .create(<Header title='This is header' />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});