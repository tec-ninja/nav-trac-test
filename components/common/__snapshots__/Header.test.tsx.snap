// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="w-full px-5 py-2 bg-blue-600 text-white text-2xl"
>
  This is header
</div>
`;
